python/orca/tutorial/NCF/utils.py

```diff
@@ -117,6 +117,15 @@ def schedule_func(epoch, lr):
         return lr * 0.5
 
 
+def get_pytorch_callbacks(args):

```

Comment:
change args to tensorboard and directly input args.tensorboard?

Reply:
Line 123 needs `args.model_dir`
so passing `args` is more simplified?

Reply:
okay, make sense